2101/no jira fix to add statement defn (#46)
Invoca/declare_schema: CHANGELOG.md

Note: this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).

## [0.6.2] - 2021-01-06

> ### Changed
> - added compatibility params to DeclareSchema::Model::ForeignKeyDefinition#to_add_statement

Reviewer: ```suggestion
### Fixed
- Fixed a but in migration generation caused by `DeclareSchema::Model::ForeignKeyDefinition#to_add_statement`
not being passed proper arguments
```

> ### Added
> - sqlite3 as dev dependency

Reviewer: ```suggestion
### Added
- Added `sqlite3` as dev dependency for local development
```

### Added
- Initial version from https://github.com/Invoca/hobo_fields v4.1.0.

> [0.6.1]: https://github.com/Invoca/declare_schema/compare/v0.6.1...v0.6.2

Reviewer 2: ```suggestion
[0.6.2]: https://github.com/Invoca/declare_schema/compare/v0.6.1...v0.6.2
```
